packages/core/src/interpreter.ts

```diff
@@ -991,7 +992,7 @@ export class Interpreter<
           return;
         } else {
           if (sendAction.to) {
-            this.sendTo(sendAction._event, sendAction.to);
+            this.sendTo(sendAction._event, sendAction.to, _event === initEvent);
```

Comment:
Was the problem that it was not sent immediately, or not sent at all?

Reply:
they would be flushed within the next `service.send(ev)` - which could happen at any time, or not at all.